src/components/VAudioTrack/VPlayPause.vue

```diff
@@ -69,12 +70,27 @@ export default defineComponent({
      * Get the button icon based on the current status of the player.
      */
     const icon = computed(() => statusIconMap[props.status])
+
+    const buttonProps = computed(() => {
+      const connections = (
+        props.layout === 'row'
+          ? 'end'
+          : props.layout === 'global'
+          ? 'all'
+          : 'none'
```

Comment:
The chained ternaries are a tad bit hard to read.

Reply:
I don't like it either, extracted it to a mapping object.